.cargo/config

```diff
@@ -1,6 +1,12 @@
 [alias]
 xtask = "run --package xtask --"
 
+# Faster linking in CI for doctests where lld is available
```

Comment:
By putting it here instead of injecting it only in the CI, don't we force anybody building PyO3 locally to have LLD available as well? Or is it shipped with rustup nowadays?

Reply:
Hmm, great question. Unfortunately I'm running on macOS so can't test that.

Anyway, it looks like putting this here breaks netlify, so, I guess we should put it in the GitHub Actions. I'll take a look at  doing so...